apps/content_types/content_types.py

```diff
@@ -253,12 +265,21 @@ def get_fields_map_and_names():
     field_names = {}
 
     for vocabulary in vocabularies:
+        if vocabulary.get('selection_type') == DO_NOT_SHOW_SELECTION:
+            continue
         fields_map[vocabulary.get('schema_field', vocabulary['_id'])] = vocabulary['_id']
         field_names[vocabulary['_id']] = vocabulary.get('display_name', vocabulary['_id'])
 
     return fields_map, field_names
 
 
+def get_forbiden_vocabularies():
+    vocabularies = get_resource_service('vocabularies').get_custom_vocabularies()
+
+    return (vocabulary for vocabulary in vocabularies
```

Comment:
imo we can filter it via db, that might be quicker than fetching all vocabularies with items

Reply:
Done